pyproject.toml

```
@@ -249,3 +270,59 @@ reportMissingParameterType = false  # 4000+ errors
 reportUnknownVariableType = false  # 6000+ errors
 reportUnknownArgumentType = false  # 6000+ errors
 reportUnknownMemberType = false  # 20000+ errors
+
+[tool.uv]
+default-groups = [
+	"dev",
+	"docs",
+	"lint",
+]
+package = false
+python-preference = "only-managed"
+environments = ["sys_platform == 'win32'"]
+
```

Consider providing the NVDA packages/modules for projects depending on NVDA, like add-ons that want to depend on NVDA to create a development environment.

```suggestion
python-preference = "only-managed"
environments = ["sys_platform == 'win32'"]

[tool.setuptools.packages.find]
where = ["source"]  # list of folders that contain the packages (["."] by default)

```